Crash when launching a backup session with volume snapshot

Hi,

Got this error in the job that is implementing the backup session:

```
 kubectl logs stash-vs-deploy-my-awesome-deployment-session-6j8wc
I0405 21:09:43.929486       1 log.go:172] FLAG: --alsologtostderr="false"
I0405 21:09:43.930922       1 log.go:172] FLAG: --backupsession="my-awesome-deployment-session"
I0405 21:09:43.930933       1 log.go:172] FLAG: --bypass-validating-webhook-xray="false"
I0405 21:09:43.930941       1 log.go:172] FLAG: --enable-analytics="false"
I0405 21:09:43.930970       1 log.go:172] FLAG: --help="false"
I0405 21:09:43.930978       1 log.go:172] FLAG: --kubeconfig=""
I0405 21:09:43.930987       1 log.go:172] FLAG: --log-flush-frequency="5s"
I0405 21:09:43.930998       1 log.go:172] FLAG: --log_backtrace_at=":0"
I0405 21:09:43.931015       1 log.go:172] FLAG: --log_dir=""
I0405 21:09:43.931024       1 log.go:172] FLAG: --logtostderr="true"
I0405 21:09:43.931031       1 log.go:172] FLAG: --master=""
I0405 21:09:43.931038       1 log.go:172] FLAG: --metrics-enabled="true"
I0405 21:09:43.931055       1 log.go:172] FLAG: --pushgateway-url="http://stash.stash.svc:56789"
I0405 21:09:43.931063       1 log.go:172] FLAG: --service-name="stash-operator"
I0405 21:09:43.931071       1 log.go:172] FLAG: --stderrthreshold="0"
I0405 21:09:43.931078       1 log.go:172] FLAG: --target-kind="Deployment"
I0405 21:09:43.931085       1 log.go:172] FLAG: --target-name="my-awesome-deployment"
I0405 21:09:43.931093       1 log.go:172] FLAG: --use-kubeapiserver-fqdn-for-aks="true"
I0405 21:09:43.931100       1 log.go:172] FLAG: --v="3"
I0405 21:09:43.931108       1 log.go:172] FLAG: --vmodule=""
W0405 21:09:43.933440       1 client_config.go:541] Neither --kubeconfig nor --master was specified.  Using the inClusterConfig.  This might not work.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x16679f9]

goroutine 1 [running]:
stash.appscode.dev/stash/pkg/util.WaitUntilVolumeSnapshotReady.func1(0xc0008a6d40, 0x1391d4d, 0x22fc4c0)
	/src/pkg/util/kubernetes.go:450 +0xc9
k8s.io/apimachinery/pkg/util/wait.pollImmediateInternal(0xc000710ce0, 0xc0008a6da8, 0xc000710ce0, 0x7f27bb144008)
	/src/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:338 +0x2b
k8s.io/apimachinery/pkg/util/wait.PollImmediate(0x2faf080, 0x68c61714000, 0xc0008a6da8, 0xc0008a6dc8, 0x40cb48)
	/src/vendor/k8s.io/apimachinery/pkg/util/wait/wait.go:334 +0x4d
stash.appscode.dev/stash/pkg/util.WaitUntilVolumeSnapshotReady(0x2a3eca0, 0xc0007b4c60, 0xc000710ca0, 0x19, 0x0, 0x0, 0xc000526c99, 0x7, 0xc00070a070, 0x62, ...)
	/src/pkg/util/kubernetes.go:448 +0x7e
stash.appscode.dev/stash/pkg/cmds.(*VSoption).createVolumeSnapshot(0xc000538000, 0xc00034c520, 0x1d, 0x0, 0x0, 0xc000331380, 0x7, 0xc000099680, 0x60, 0xc00082a990, ...)
	/src/pkg/cmds/create_volumesnapshot.go:183 +0x258
stash.appscode.dev/stash/pkg/cmds.NewCmdCreateVolumeSnapshot.func1(0xc00074a280, 0xc000575080, 0x0, 0xb, 0x0, 0x0)
	/src/pkg/cmds/create_volumesnapshot.go:105 +0x4ca
github.com/spf13/cobra.(*Command).execute(0xc00074a280, 0xc000574fd0, 0xb, 0xb, 0xc00074a280, 0xc000574fd0)
	/src/vendor/github.com/spf13/cobra/command.go:826 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0xc00021d900, 0x2, 0x0, 0x0)
	/src/vendor/github.com/spf13/cobra/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/src/vendor/github.com/spf13/cobra/command.go:864
main.main()
	/src/main.go:40 +0x8d
```

Stash version is v0.9.0-rc.6.

On the cloud provider (Scaleway), the volume snapshot have been created. It's also displayed as a volumesnapshot resource in Kubernetes:

```
kubectl get volumesnapshot
NAME                        READYTOUSE   SOURCEPVC           SOURCESNAPSHOTCONTENT   RESTORESIZE   SNAPSHOTCLASS   SNAPSHOTCONTENT                                    CREATIONTIME   AGE
scaleway-pvc-test-session   true         scaleway-pvc-test                           1Gi           scw-snapshot    snapcontent-eb3f890e-91f9-406a-8289-8124dcc9c638   10h            10h

kubectl get volumesnapshotcontent
NAME                                               READYTOUSE   RESTORESIZE   DELETIONPOLICY   DRIVER             VOLUMESNAPSHOTCLASS   VOLUMESNAPSHOT              AGE
snapcontent-eb3f890e-91f9-406a-8289-8124dcc9c638   true         1073741824    Delete           csi.scaleway.com   scw-snapshot          scaleway-pvc-test-session   10h
```

--- a/pkg/util/kubernetes.go
+++ b/pkg/util/kubernetes.go
@@ -447,7 +447,7 @@ func WaitUntilDeploymentConfigReady(c oc_cs.Interface, meta metav1.ObjectMeta) e
 func WaitUntilVolumeSnapshotReady(c snapshot_cs.Interface, meta metav1.ObjectMeta) error {
 	return wait.PollImmediate(apis.RetryInterval, 2*time.Hour, func() (bool, error) {
 		if obj, err := c.SnapshotV1beta1().VolumeSnapshots(meta.Namespace).Get(meta.Name, metav1.GetOptions{}); err == nil {
-			return obj.Status.ReadyToUse != nil && *obj.Status.ReadyToUse, nil
+			return obj.Status != nil && obj.Status.ReadyToUse != nil && *obj.Status.ReadyToUse, nil
 		}
 		return false, nil
 	})

